components/raftstore-v2/src/router/mod.rs

```diff
@@ -2,14 +2,15 @@
 
 mod imp;
 mod internal_message;
-mod message;
-mod response_channel;
+pub mod message;
```

Comment:
Should not be public.

Reply:
every message is to be consumed by other module's code. You have to pub use every single definition in message if it's not pub.